src/rpdk/core/contract/resource_client.py

```diff
@@ -195,19 +203,27 @@ def update_strategy(self):
         return self._update_strategy
 
     def generate_create_example(self):
+        if self._inputs:
+            return self._inputs[0]
         example = self.strategy.example()
         return override_properties(example, self._overrides.get("CREATE", {}))
 
     def generate_invalid_create_example(self):
+        if self._inputs:
+            return self._inputs[2]
```

Comment:
Will there always be 3 inputs?

Reply:
yes, I expect to get create, update and invalid inputs from the user